VolunteerApp/Models/Category.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+
+namespace VolunteerApp.Models
+{
+    public class Category
+    {
+        [Key]
```

Comment:
https://www.entityframeworktutorial.net/efcore/fluent-api-in-entity-framework-core.aspx

Reply:
fixed